src/bootstrap/src/core/config/flags.rs

```diff
@@ -133,6 +133,13 @@ pub struct Flags {
     /// whether to use color in cargo and rustc output
     pub color: Color,
 
+    #[arg(global(true), long)]
+    /// Bootstrap uses this value to decide whether it should lock or bypass locking its build directory.
```

Comment:
I guess "build directory" here feels misleading. We are using this to protect the *entire* build -- right? In particular, it is not safe to pass this option just because you've already built rustbuild itself.

I'd feel better removing the "directory" phrasing here and in 1-2 other places.

Reply:
I agree, "its build directory" feels like directory for the bootstrap artifacts rather than compiler/std/tool ones.